libraries/OPAMP/README.md

```diff
@@ -0,0 +1,58 @@
+# OPAMP Libray for Uno R4 Minima & WiFi
+
+## Description
+
+The Arduino Uno R4 Minima and WiFi boards, both a featuring a Renesas R7FA4M1AB3CFM#AA0 microcontroller, do have a built-in OPAMP peripheral.
+
+OMAMPs are very versatile. They can:

```

Comment:
Typo indeed. Should the library folder and header itself be renamed from `OPAMP` to something else too or is this fine?

Reply:
I'm fine with your use of "OPAMP" elsewhere. I see that is what is used in the RA4M1 datasheet. I only thought it would be helpful to also provide the user with the "operational amplifier" and "op amp" keywords in the introduction.